Jenkinsfile.integration

```diff
@@ -50,16 +36,16 @@ node('lisk-integration') {
 			try {
 				deleteDir()
 				checkout scm
-
-				sh '''
-					# Cycle postgres, install deps and prepare config
-					sudo service postgresql restart
-					rsync -axl -e "ssh -oUser=jenkins" master-01:/var/lib/jenkins/lisk/node_modules/ "$WORKSPACE/node_modules/" || true
-					npm install
-					cp test/data/config.json test/data/genesis_block.json .
-				'''
+				// Cycle postgres, restore chache install deps and prepare config
+				sh 'sudo service postgresql restart'
+				cache_file = restoreCache("config.json")
+				sh 'npm install'
+				saveCache(cache_file, './node_modules', 7)
+				sh 'cp test/data/config.json test/data/genesis_block.json .'
```

Comment:
When this PR will be merged https://github.com/LiskHQ/lisk/pull/2181 these files will be no longer exists. So I suggest we hold this PR, until other one merged.